modules/gitops-deleting-rolloutmanager-custom-resource.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * openshift-docs/cicd/gitops/argo-rollouts-for-progressive-delivery-of-deployments.adoc
+
+:_content-type: PROCEDURE
+[id="gitops-deleting-rolloutmanager-custom-resource_{context}"]
+= Deleting a RolloutManager custom resource
+
+Uninstalling the {gitops-title} Operator does not remove the resources that were created during installation. To uninstall the progressive delivery of deployments by using {gitops-title}, you must explicitly delete the resources by deleting the `RolloutManager` custom resource (CR), and then uninstall the {gitops-title} Operator.
+
+.Prerequisites
+
+* {gitops-title} 1.9.0 or a newer version is installed in your cluster.
+* A `RolloutManager` CR exists in your namespace.
+
+.Procedure 
+
+. Log in to the {product-title} web console as a cluster administrator. 
+
+. In the *Administrator* perspective, click *Operators* -> *Installed Operators*.
+
+. Create or select the project where `RolloutManager` CR for the progressive delivery is setup from the *Project* drop-down menu.
+
+. Select *OpenShift GitOps Operator* from the installed operators.
+
+. In the Tabs section, click *RolloutManager* to find the list of RolloutManager instances under the *RolloutManagers* section.
```

Comment:
> In the Tabs section

I'm confused about this - should this be `Click the *RolloutManager* tab`? 

The "to find the list..." part also seems like it could be simplified, but I'm not familiar enough with this UI to make a suggestion here.